Studnet managment system homework

aharutyunyan9 opened the pull request:
Hello Yana and Arman jans,I have an issue with/commit push files with the terminal,so I am uploading the file directly here ..

aharutyunyan9 committed:
Studnet managment system homework

armwhite commented on Main.py:
```diff
@@ -0,0 +1,20 @@
+
+grade_for_current_year = int(input ("Please enter your grade for_the_current_year"))
+grade_for_past_year = int(input ("Please enter your grade for_past_year"))
+total_grade = (grade_for_current_year + grade_for_past_year) /2
```
This is the average grade.

armwhite commented on Main.py:
```diff
@@ -0,0 +1,20 @@
+grade_for_current_year = int(input ("Please enter your grade for_the_current_year"))
+grade_for_past_year = int(input ("Please enter your grade for_past_year"))
+total_grade = (grade_for_current_year + grade_for_past_year) /2
+print ("Total Grade:", total_grade)
```
As I told before, we need to provide informative error messages. Like adding student's name, telling that the average grade for the last two years are .... and so.

armwhite commented on Main.py:
```diff
@@ -0,0 +1,20 @@
+total_grade = (grade_for_current_year + grade_for_past_year) /2
+print ("Total Grade:", total_grade)
+if total_grade <50:
+    print ("Sorry, you are a loser")
```
Again, we have issues with messaging.

armwhite commented on Main.py:
```diff
@@ -0,0 +1,20 @@
+student_name = input ("Please enter your full name.")
+student_age = int(input ("please enter your age"))
+if student_age <18:
```
We discussed doing checks for input values

aharutyunyan9 committed:
Update Student management system part 1 

@Yana-Gabrielyan @armwhite  Hello)  added validations for inputs, also retry block in case of incorrect input, the texts are also adjusted , please approve

aharutyunyan9 committed:
This is the 4rd part
